build.gradle

```diff
@@ -22,7 +22,7 @@ plugins {
 
 group = "com.amazon.ion"
 version = "1.2.0-SNAPSHOT"
-ext.kotlin_version = "[1.3,)"
+ext.kotlin_version = "[1.3,1.5["
```

Comment:
This made me lookup what `[` [meant][1] at the end of the range, which leads me to ask is there a reason to prefer `[` over `)` for exclusive end of range or did I miss a difference for this syntax? 

[1]: https://docs.gradle.org/current/userguide/single_versions.html

Reply:
There is no difference between them, and I didn't put any thought into one syntax vs the other. Do we care enough to change it back to `)`? (I'll do it if someone prefers it that way.)